### PR TITLE
feat(plugin): enables lootboxes back again

### PR DESCRIPTION
**Pull request by happyendermangit**
Discord as of today disabled lootboxes, so i made this plugin that enables them back
image
image

### CODE REVIEW

**Review by mufaroxyz — COMMENTED** [review]
LGTM

File: src/plugins/LootBoxes/LootBoxes.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import definePlugin from "@utils/types";
+
+export default definePlugin({
+    name: "LootBoxes",
+    description: 'Enables lootboxes back.',
+    authors: [Devs.HAPPY_ENDERMAN],
+    patches: [
+        /** enable the badge */
+        {
+            find: "Messages.PACKAGES_BADGE",
```
Comment (MrDiamondDog) [inline src/plugins/LootBoxes/LootBoxes.js:29]: Dont use \w+, use \i (custom selector made by us)

Comment (Vendicated) [reply src/plugins/LootBoxes/LootBoxes.js:29]: no it isn't. variable names may contain characters that are not matched by \w, such as $

**Comment by Vendicated** [comment]
i'm not sure how useful this is.

won't discord just remove the api / code for this very soon? longevity of this plugin seems unlikely

why not post it as a js snippet instead?

**Comment by happyendermangit** [comment]
The api will never be removed, discord will just forget about this, a js snippet is not possible...

**Comment by MrDiamondDog** [comment]
It is getting removed very soon https://discord.com/channels/1015060230222131221/1105582084237570088/1227358383468838965

**Review by byeoon — CHANGES_REQUESTED** [review]
If this were to be a update that was pushed upstream, this can't be the README.

**Pull request closed without merging**